project-base/tests/App/Test/Codeception/Module/StrictWebDriver.php

```diff
@@ -239,6 +239,7 @@ public function findElementByCss(string $css): WebDriverElement
 
     /**
      * @param \Facebook\WebDriver\WebDriverElement $webDriverElement
+     * @deprecated This method is not used so it will be removed in next major version
```

Comment:
Changes in the project-base are not BC-breaks.
How about removing it with the possible replacement in the upgrade notes?

Reply:
After discussion, I have kept those methods as they can be useful + I have marked previous deprecate methods as internal as we do not intend to remove them fcb7acaaf5e70ce32607c23f5a37962ef224bdb7